account_asset_management/models/account_asset.py

```diff
@@ -1276,7 +1276,7 @@ def _xls_acquisition_fields(self):
         """
         Update list in custom module to add/drop columns or change order
         """
-        return [
+        acquisition_fields = [
```

Comment:
I think the variable name can be simplified, as the method where is declared already indicates the rest (the same for the rest):

```suggestion
        fields = [
```